db/01_create_peep_table.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TABLE users(id SERIAL PRIMARY KEY, user_name VARCHAR(30), password VARCHAR(30)); 
+ CREATE TABLE peeps(id SERIAL PRIMARY KEY, user_id INTEGER REFERENCES users(id), peep VARCHAR(140));
```

Comment:
can you split this into two files? I think each sql file needs to have one instruction